Documentation/Installation/Index.rst

```diff
@@ -18,14 +18,13 @@ Download phar archive
 
 To install Surf as phar archive, run the following commands::
 
-    mkdir /usr/local/surf
-    curl -L https://github.com/TYPO3/Surf/releases/download/|release|/surf.phar -o /usr/local/surf/surf.phar
-    chmod +x /usr/local/surf/surf.phar
-    ln -s /usr/local/surf/surf.phar /usr/local/bin/surf
+    curl -L https://github.com/TYPO3/Surf/releases/download/2.1.1/surf.phar -o surf.phar
```

Comment:
@sabbelasichon Sphinx is driving me crazy here as I didn't found any solution how to properly add a placeholder in here. Using `.. parsed-literal::` produces this: 
![image](https://user-images.githubusercontent.com/941794/84961966-e0a3b780-b105-11ea-85ec-4b7db1981c74.png) :cry: